digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/Digiroad2Api.scala

```diff
@@ -607,13 +611,13 @@ class Digiroad2Api(val roadLinkService: RoadLinkService,
       "maxAddressNumberRight" -> roadLink.attributes.get("TO_RIGHT"),
       "minAddressNumberLeft" -> roadLink.attributes.get("FROM_LEFT"),
       "maxAddressNumberLeft" -> roadLink.attributes.get("TO_LEFT"),
-      "roadPartNumber" -> extractLongValue(roadLink, "VIITE_ROAD_PART_NUMBER"),
-      "roadNumber" -> extractLongValue(roadLink, "VIITE_ROAD_NUMBER"),
+      "roadPartNumber" -> roadLink.attributes.getOrElse("VIITE_ROAD_PART_NUMBER", roadLink.attributes.get("TEMP_ROAD_PART_NUMBER")),
```

Comment:
The extractLongValue provides an Option[Long]. Are we always guaranteed VIITE_ROAD_PART_NUMBER or TEMP_ROAD_PART_NUMBER in these cases?

Reply:
The goal was to guarantee that whenever there's no information from VIITE side, we would deliver the information from this Temporary Road information, so yes.